tests/tools/sessionvalidation/sessionvalidation.py

```diff
@@ -220,9 +223,9 @@ def validateSingleTransaction(txn):
             # errors by definition can't have bodies
             response_line = txn_resp.getHeaders().split('\r\n')[0]
             response_code = response_line.split(' ')[1]
-            if response_code.startswith('3') and txn_resp.getBody():
-                _verbose_print("transaction response was 3xx and had a body")
-                retval = False
+            # if response_code.startswith('3') and txn_resp.getBody():
+            #     _verbose_print("transaction response was 3xx and had a body")
+            #     retval = False
```

Comment:
Why is this commented out?

Reply:
Yes, you should either remove,  add back, or put some comment about why this code is questionable.